docs/WSL.md

```diff
@@ -21,31 +21,16 @@ here: https://channel9.msdn.com/Shows/Tabs-vs-Spaces/GPU-Accelerated-Machine-Lea
 
 Requirements: Windows 10 version 2004 or higher
 
-The instructions are [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10), but summarized in
-copy/paste-able form below. When installing via the UI, pick Ubuntu version 20.04 LTS as your distribution.
-
 To use the commandline setup, please first install
 [winget via the appxbundle](https://github.com/microsoft/winget-cli/releases).
 
-Then, in PowerShell as Administrator:
-
-```
-dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
-dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
-```
-
-Restart your machine, then again in PowerShell:
+Optionally, restart your machine and in PowerShell as Administrator type:
```

Comment:
I find it a bit confusing, unclear if the "optionally" applies to the whole sentence?
```suggestion
- Optionally, restart your machine
- In PowerShell as Administrator type:
```

Reply:
Just split the sentence to make it more clear